javascript/javascript2/week3/index.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
+console.log("JS2 WEEK3");
+
+// Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+window.addEventListener("DOMContentLoaded", function (e) {
+  setTimeout(() => {
+    console.log("Called after 2.5 seconds after script load");
+  }, 2500);
+});
+
+// Create a function that takes 2 parameters: delay and stringToLog. Calling this function should log out the stringToLog after delay seconds. Call the function you have created with some different arguments.
+function delayLog(delay, stringToLog) {
+  setTimeout(function () {
+    console.log(stringToLog);
+  }, delay);
+}
+
+delayLog(3000, "This string logged after 3 seconds");
+
+// Create a button in html. When clicking this button, use the function you created in the previous task to log out the text: Called after 5 seconds 5 seconds after the button is clicked.
+const button5Second = document.querySelector("#called");
+button5Second.addEventListener("click", function () {
+  delayLog(3000, "called after 5 seconds");
+});
+
+// Create two functions and assign them to two different variables. One function logs out Earth, the other function logs out Saturn. Now create a new third function that has one parameter: planetLogFunction. The only thing the third function should do is call the provided parameter function. Try call the third function once with the Earth function and once with the Saturn function.
+function earth() {
+  const var1 = "";
+  console.log("Earth");
+}
+function saturn() {
+  const var2 = "";
+  console.log("Saturn");
+}
+const planetLogFunction = (logFunction) => {
+  logFunction();
+};
+planetLogFunction(earth);
+planetLogFunction(saturn);
+
+// Create a button with the text called "Log location". When this button is clicked the location (latitude, longitude) of the user should be logged out using this browser api
+const btnLocation = document.querySelector(".location");
```

Comment:
better change 'location' to id. Class names are generally for css and id for javascript